plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js

```diff
@@ -36,7 +36,7 @@ const MultiContainerFormAdvancedSection = ({ data, path }) => {
         Advanced Settings
       </Trans>
       <Trans render="p">Advanced settings of the container.</Trans>
-      <p>{getForcePullSection(data, path)}</p>
+      <div className="form-group">{getForcePullSection(data, path)}</div>
```

Comment:
Is this related to file-based secrets?

Reply:
Not at all, but CI is failing due to prettier changes not being conform, therefore I added those as a separate commit

Reply:
Oh wait those are different ones, I am unsure how they landed here, let me check

Reply:
Must have been a wrongly resolved merge conflict, I'll recreate the branch once everything is working properly

Reply:
Cool.